src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocCorrect.java

```diff
@@ -153,7 +153,7 @@ void foo6() {}
         boolean emulated(String s) {return false;}
 
         /**
-         * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}.
+         * As of JDK 1.1, replaced by .
```

Comment:
please revert all changes that are not related to target of issue we are fixing.

Reply:
Done

Reply:
Done